src/routes/Main/ApisList/index.tsx

```diff
@@ -107,6 +111,14 @@ const ApisList = () => {
                     filterQuery += " and ";
                 }
             });
+            if (filterQuery.length > 0) {
+                filterQuery = "$filter=" + filterQuery;
+                if (config.scopingFilter.length > 0) {
+                    filterQuery += " and " + config.scopingFilter;    
```

Comment:
Do we need to wrap this in parentheses to preserve the boolean logic? Otherwise the boolean algebra can get messed up.

Reply:
It's always appended in the end. So I think that even if you have grouped filters with parentheses this will work.